test/common/router/header_formatter_test.cc

```diff
@@ -1666,23 +1785,26 @@ match: { prefix: "/new_endpoint" }
 
   {
     Http::TestResponseHeaderMapImpl header_map;
-    resp_header_parser->evaluateHeaders(header_map, stream_info);
+    resp_header_parser->evaluateHeaders(
+        header_map, *Http::StaticEmptyHeaders::get().request_headers, header_map, stream_info);
```

Comment:
ditto?

Reply:
Expanded the test to make response headers to be evaluated dependent on both request and response headers. Therefore `evaluateHeaders` must be called with all 4 params.